centreon-open-tickets/widgets/open-tickets/src/export.php

```diff
@@ -198,43 +203,106 @@
     }
 }
 
-if (isset($preferences['downtime_filter']) && $preferences['downtime_filter']) {
+if (! empty($preferences['downtime_filter'])) {
     if ($preferences['downtime_filter'] == "downtime") {
         $query = CentreonUtils::conditionBuilder($query, " s.scheduled_downtime_depth > 0 ");
     } elseif ($preferences['downtime_filter'] == "ndowntime") {
         $query = CentreonUtils::conditionBuilder($query, " s.scheduled_downtime_depth = 0 ");
     }
 }
 
-if (isset($preferences['state_type_filter']) && $preferences['state_type_filter']) {
+if (! empty($preferences['state_type_filter'])) {
     if ($preferences['state_type_filter'] == "hardonly") {
         $query = CentreonUtils::conditionBuilder($query, " s.state_type = 1 ");
     } elseif ($preferences['state_type_filter'] == "softonly") {
         $query = CentreonUtils::conditionBuilder($query, " s.state_type = 0 ");
     }
 }
 
-if (isset($preferences['hostgroup']) && $preferences['hostgroup']) {
+if (! empty($preferences['poller'])) {
+    $resultsPoller = explode(',', $preferences['poller']);
+    $queryPoller = '';
+    foreach ($resultsPoller as $resultPoller) {
+        if ($queryPoller != '') {
+            $queryPoller .= ', ';
+        }
+        $queryPoller .= ':instance_id_' . $resultPoller;
+        $mainQueryParameters[] = [
+            'parameter' => ':instance_id_' . $resultPoller,
+            'value' => (int)$resultPoller,
+            'type' => PDO::PARAM_INT
+        ];
+    }
+    $instanceIdCondition = ' i.instance_id IN (' . $queryPoller . ')';
```

Comment:
you need to apply the modifications that you've made in the index.php inside the export.php file